code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -407,3 +408,9 @@
 		return attack_hand(user)
 
 
+/turf/simulated/wall/AltClick(mob/user)
+	if(isliving(user))
+		var/mob/living/livingUser = user
+		if(try_graffiti(livingUser, livingUser.get_active_hand()))
+			return
+	. = ..()
```

Comment:
```suggestion
```

Alt-clicking to view tile contents is a long standing and dramatically more useful behavior. Intercepting it this way is a mistake.